omero/users/cli/index.txt

```diff
@@ -5,6 +5,11 @@ The |CLI| is a set of Python_ based system-administration, deployment and
 advanced user tools. Most of commands work remotely so that the |CLI| can be
```

Comment:
This was not changed in this PR, nevertheless, the brackets around the ``(CLI)`` seem out of place here.